tablecodec/tablecodec_test.go

```diff
@@ -151,7 +151,9 @@ func (s *testTableCodecSuite) TestTimeCodec(c *C) {
 	row := make([]types.Datum, colLen)
 	row[0] = types.NewIntDatum(100)
 	row[1] = types.NewBytesDatum([]byte("abc"))
-	ts, err := types.ParseTimestamp(nil, "2016-06-23 11:30:45")
+	ts, err := types.ParseTimestamp(&stmtctx.StatementContext{
```

Comment:
It's not stable to use local time zone in the test code.

Reply:
Done. fixed them to UTC